docs/GettingStartedDocs/UseNonDCAPDriverInAcc.md

```diff
@@ -0,0 +1,53 @@
+# Using the Non-DCAP Driver in an ACC VM
+
+ACC VMs come pre equipeed with the azure DCAP stack preinstalled. This can interfere with non-azure dcap clients, 
```

Comment:
typo "equipeed"
Also "azure" should be capitalized
Also expand acronyms (ACC, DCAP) on first use in the file.